event-stream-service/charts/event-stream-service/templates/nginx-route.yaml

```
@@ -0,0 +1,22 @@
+{{- if .Values.ess.nginx.route.enabled -}}
+{{- $routeName := (.Values.ess.nginx.route.name) }}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: {{ $routeName }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+  annotations:
+    haproxy.router.openshift.io/timeout: {{ .Values.ess.nginx.route.timeout }}
```

To add rate limiting to this you would do:
```suggestion
  annotations:
    haproxy.router.openshift.io/rate-limit-connections: 'true'
    haproxy.router.openshift.io/rate-limit-connections.concurrent-tcp: {{ .Values.ess.route.concurrent-tcp }}
    haproxy.router.openshift.io/timeout: {{ .Values.ess.nginx.route.timeout }}
```

See [FORMS-1630](https://bcdevex.atlassian.net/browse/FORMS-1630) for details.